test/httpwstest.cpp

```diff
@@ -164,29 +158,15 @@ void HTTPWSTest::testSaveOnDisconnect()
                            socket1->waitForDisconnection(std::chrono::seconds(5)));
         LOK_ASSERT_MESSAGE("Expected successful disconnection of the WebSocket 2",
                            socket2->waitForDisconnection(std::chrono::seconds(5)));
-    }
-    catch (const Poco::Exception& exc)
-    {
-        LOK_ASSERT_FAIL(exc.displayText());
```

Comment:
Why don't we fail on exception anymore ? that seems important.

Reply:
It's hard to tell in the diff, but this was two sequential try blocks that I combined into one.